plex_trakt_sync/main.py

```diff
@@ -189,16 +189,21 @@ def process_show_section(s, watched_set, listutil):
     logging.info("Now working on show section {} containing {} elements".format(s.title, len(allShows)))
     for show in allShows:
         guid = show.guid
+        if guid.startswith('plex://show/'):
+            if len(show.guids) > 0:
+                logging.debug("trying first alternative guid: " + str(show.guids[0].id))
+                guid = show.guids[0].id
```

Comment:
Please share what are the values of `show.guids`

Reply:
I don't know how to answer this, I'm sorry